### ISSUE TITLE
Your proxy directory must be writable

### ISSUE THREAD

**Issue by mgscreativa** [coolcsn/CsnUser#12]
Hi, after manual enabling a new registered user (because email confirmation didn't work), I get this on login submit "Your proxy directory must be writable"

Tried creating directory data/DoctrineORMModule/Proxy in my SkeletonApp root, but problem persists.

Using ZF 2.3.0, CsnUser git latest, and complete fresh install of SkeletonApp all through composer

**Comment by mgscreativa** [comment]
Well, rechecked permissions on data/DoctrineORMModule/Proxy and got tr working.

I think you must add a step for creating that directory in the readme

**Comment by coolcsn** [comment]
Yes, we were thinking abot addin in the documentaiotn that and keep
forgeting. :)

**Comment by mgscreativa** [comment]
Don't worry, I'm following video "ZF2 modules coolcsn. Git branching model. Semantic Versioning. How to use or participate", so I expect to do some pull requests this week...
